runtime/doc/usr_21.txt

```diff
@@ -255,15 +255,18 @@ well stand for "source").
 The windows that were open are restored, with the same position and size as
 before.  Mappings and option values are like before.
    What exactly is restored depends on the 'sessionoptions' option.  The
-default value is "blank,buffers,curdir,folds,help,options,winsize".
+default value is:
+"blank,buffers,curdir,folds,help,options,tabpages,winsize,terminal".
```

Comment:
Should this mention the Nvim default (without "options") instead? It didn't before.